node/resolvers/search/index.ts

```diff
@@ -558,7 +558,17 @@ export const queries = {
       tradePolicy,
     })
 
-    return result
+    const productResolver = args.productOriginVtex
+      ? productsCatalog
+      : productsBiggy
+
+    const convertedProducts = productResolver(
+      result,
+      ctx,
+      args.simulationBehavior
```

Comment:
don't you need to add a parameter for this in the `productsCatalog`?

Reply:
Done!